package.json

```diff
@@ -12,7 +12,8 @@
     "license": "lerna exec -- cp ../../LICENSE .",
     "release": "yarn license && lerna publish from-package",
     "serve": "serve packages/vega/ -l 8080",
-    "test": "lerna run test && yarn lint",
+    "test": "yarn run test:no-lint && yarn run lint",
```

Comment:
I believe this should be `yarn test:no-lint && yarn lint`. No `run` required.

Reply:
You are right, I didn't realize that the run was optional.